README.md

```diff
@@ -13,16 +13,18 @@ var cleankill = require('cleankill');
 To register a handler:
 
 ```js
-cleankill.onInterrupt(function(done) {
-  // do things.
-  done();
+cleankill.onInterrupt(() => {
```

Comment:
I find that hard to read and easy to miss the fact that it's in a promise body. IMO if the arrow function body is longer than one line it should be a full block statement.